fontforgeexe/cvgetinfo.c

```diff
@@ -62,6 +62,7 @@ typedef struct gidata {
     GGadgetCreateData* gcd;
     GGadget *group1ret, *group2ret;
     int nonmodal;
+    void (*closefunc)(struct gidata *);
```

Comment:
Not a fan of this, adding a function pointer just for this introduces too much indirection.

Since it's not used any more, rename the `nonmodal` flag to be `is_spiro` or something like that, and just do a conditional on that in the close event